### ISSUE TITLE
Ligatures broken with 24-bit color in Neovim

### ISSUE THREAD

**Issue by kybaeus** [zeit/hyper#1308]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

Running on the latest version and up-to-date archlinux, ligatures do not work if neovim has `set termguicolors` enabled.

**Comment by gerred** [comment]
I have this as well, on MacOS Sierra with the latest Hyper.app and neovim 0.1.7.

**Comment by benatkin** [comment]
This seems like something that might have unintentionally been fixed since then. I'm going to test it out and report back.

**Comment by benatkin** [comment]
I configured Hyper to use FuraCode Nerd Font. I found that ligatures were not enabled by default. Installing [hyper-ligatures](https://github.com/timneutkens/hyper-ligatures) got them working, but in Neovim they would sometimes not work. For instance, if I went to a new JavaScript tab and typed `<=` the ligature would not appear right away, but when I left the tab and went back to it, the ligature was there. I tried disabling 24 bit color in Neovim and still had the same issue. So, ligatures aren't supported by default, but they work sometimes with the hyper-ligatures plugin, and doesn't seem to matter whether 24 bit color is enabled in Neovim.
